### DISCUSSION TITLE
Tab Icon Farbe v3.1.0

### DISCUSSION THREAD

**Discussion by Bernd9000** [Zefau/ioBroker.jarvis#1909] [Q&A] [answered]
Bis jarvis Version v3.0.13 konnte man u.a. die Icon Farbe der Tabs direkt in der Konfiguration auswählen.
In der Version v3.1.0 fehlt das bis jetzt.
Kommt das wieder rein oder wird das  jetzt über css eingestellt?

![Unbenannt](https://user-images.githubusercontent.com/77005685/192164161-fe0252d8-d9c4-4957-99a5-12b7005a8933.jpg)

**Comment by Zefau** [comment, marked as answer]
es gibt dort nun ein JSON für die Seiten bzw. Tabs, mit denen du die Farbe(n) steuern kannst.

**Reply by Bernd9000** [reply]
Ja, Danke :-), hab ich übersehen....
